Gemfile.lock

```diff
@@ -410,10 +410,10 @@ GEM
       activemodel (>= 6.0.0)
       bindex (>= 0.4.0)
       railties (>= 6.0.0)
-    webdrivers (5.2.0)
+    webdrivers (5.3.1)
       nokogiri (~> 1.6)
       rubyzip (>= 1.3.0)
-      selenium-webdriver (~> 4.0)
+      selenium-webdriver (~> 4.0, < 4.11)
```

Comment:
Just out of curiosity, did you change this because "rake test" wasn't working?

Reply:
I need to update the webdriver because the tests couldn't run:
brew install chromedriver
bundle update webdrivers